src/Entity/TaskCollection.php

```diff
@@ -128,11 +129,21 @@ public function removeTask(Task $task)
         }
     }
 
+    /**
+     * @return Task[]
+     */
     public function getTasks()
     {
-        return $this->getItems()->map(function (TaskCollectionItem $item) {
-            return $item->getTask();
-        })->toArray();
+        return $this->getItems()
+            ->map(fn($item) => $item->getTask())
+            ->toArray();
+    }
+
+    public function getActiveTasks()
```

Comment:
Maybe it would be more reusable & "expressive" to implement this via a `filterTasks` method, that accepts an expression language function. For example, it would be: 

```php 
$delivery->filterTasks('not task.cancelled');
```